src/main/java/edu/ithaca/dragon/blackjack/BlackJack.java

```diff
@@ -115,4 +125,24 @@ public static int assessHand(Hand h){
         }
         return total;
     }
+
+    public void takeDealerTurn(){
+        while (assessHand(dealer) < 17 && deck.getDeck().size() > 0){
+            dealer.addCard(deck.getNextCard());
+        }
+    }
+
+    public WinState stay(){
+        if(assessHand(hands.get(0)) > 21) return WinState.LOSE;
+        takeDealerTurn();
+        if(assessHand(dealer) > 21) return WinState.WIN;
+        int win = compareHands(hands.get(0), dealer);
+        if(win == 0) return WinState.TIE;
+        return win > 0 ? WinState.LOSE:WinState.WIN;
+    }
+
+    public static int compareHands(Hand rhs, Hand lhs){
```

Comment:
Any reason not to just return a WinState from here rather than an int?

Reply:
Down the road if we have more than one player we might be comparing hands that aren't necessarily the winning hand.